src/import-utils.ts

```diff
@@ -217,6 +237,37 @@ export function rewriteAstBaseUrl(
 }
 
 /**
+ * Walk through inline scripts of an import document.
+ * For each script create identity source maps unless one already exists.
+ *
+ * The generated script mapping detail is the relative location within
+ * the script tag. Later this will be updated to account for the
+ * line offset within the final bundle.
+ */
+export async function addSourcemapsToInlineScripts(
+    document: Document, ast: ASTNode, oldBaseUrl: UrlString) {
+  const inlineScripts = dom5.queryAll(ast, matchers.inlineJavascript);
+  const promises = inlineScripts.map(scriptAst => {
+    let content = dom5.getTextContent(scriptAst);
+    const scriptLocation =
+        (scriptAst.__location as parse5.ElementLocationInfo).startTag;
```

Comment:
What I need out of that though is a line/col number for the end of the `<script>` tag. Any way to obtain that from the analyzer?